### ISSUE TITLE
Can i use?

### ISSUE THREAD

**Issue by lassejlv** [rule-kells/qr-code-component#1]
Its is okay if i use [Tailwind CSS](https://tailwindcss.com)

**Comment by rule-kells** [comment]
Yeah, go for it! Feel free to share it when you are finished. I have not dabbled too much with Tailwind. Hope to get into it more.
